bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -378,7 +406,7 @@ async function syncEnvelopes(client, dataStore, account_id, userids, latestSynch
         dataStore.save('envelopes', templateData)
 
         start_position += COUNT;
-        totalSetSize = json['totalSetSize'];
+        totalSetSize = responseBody['totalSetSize'];
```

Comment:
```suggestion
        totalSetSize = responseBody.totalSetSize;
```

Reply:
Changes are done in code as per suggestion. 
 start_position += COUNT;
        totalSetSize = responseBody.totalSetSize;
        if (start_position > totalSetSize || totalSetSize === 0) {
            i++;
            start_position = 0
            totalSetSize = 0
        }